src/inet/InetInterface.cpp

```diff
@@ -224,6 +224,133 @@ void CloseIOCTLSocket(void)
 
 #if CHIP_SYSTEM_CONFIG_USE_SOCKETS
 
+#if __ANDROID__ && __ANDROID_API__ < 24
+
+static struct if_nameindex * backport_if_nameindex(void);
+static void backport_if_freenameindex(struct if_nameindex *);
+
+static void backport_if_freenameindex(struct if_nameindex * inArray)
+{
+    if (inArray == NULL)
+        return;
```

Comment:
Please use curly brackets for all `if()` statements

Reply:
Done.